### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by jacobosaa** [Anura3D/Anura3D_OpenSource#129] [Q&A] [answered]
Hi!

has anyone had this problem? how can i fix it? 
![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/128545000/75354933-e1f9-47bd-a01e-568f47352932)

**Comment by ayerrocolom** [comment, marked as answer]
Hello! It seems you are using an older version of Anura3D that is not compatible with the current version of GiD. You need to download the latest version of Anura3D code (v2023).

**Comment by jacobosaa** [comment]
Hello Alba! Hoping that this email finds you well, I have some issues
working with anura (as usual), and I've made an entry on the GitHub
discussion forum, but with no response. I apologize if I overstep the
regular conduct contacting you directly. Let me explain the issue.

I'm currently studying the flow behavior of thickened tailings, modeling a
different set of columns collapsing due to its own weight, comparing the
results between modelling the material as a liquid and as a saturated soil.
I'm using the 16.0.6 GiD version, with the Anura3D v.2023, using the MPM
method. I've just modeled the column using the Mohr-Coulomb constitutive
model as a saturated fully coupled soil, and there was no problem during
the calculation process. BUT, when it comes to using the Bingham
constitutive model, this message appears and I'm not able to continue with
the calculation process.
I've already checked if the .dll (A3DBingham.dll) file was in the
problemtype folder, which came "naturally" after compiling the code from
this forum one week ago. I was using an older version of Anura3D and GiD
but then opted for the newest version to get rid of this problem, which
happened to me before.

[image: image.png]

This message appears when it comes to run the exe file through CMD after
generating the calculation files. is it because the dll file has a
different name? or maybe the gom file has a line in the code that shouldn't
be there?

Hoping to read from you soon, and beforehand, thank you very much.

Jacobo
